test/parallel/test-net-server-simultaneous-accepts-produce-warning-once.js

```diff
@@ -1,12 +1,16 @@
 'use strict';
 
+// This test checks that `_setSimultaneousAccepts` does not emit DEP0121 warning
+// in second calling, by combination with
+// `test-net-deprecated-setsimultaneousaccepts.js`.
```

Comment:
```suggestion
// Test that DEP0121 is emitted only once if _setSimultaneousAccepts() is called
more than once.
```

Reply:
Or if we want the fuller explanation maybe this?

```suggestion
// Test that DEP0121 is emitted only once if _setSimultaneousAccepts() is called
// more than once. This test is similar to 
// test-net-deprecated-setsimultaneousaccepts.js, but that test calls
/// _setSimultaneousAccepts() only once. Unlike this test, that will confirm
// that the warning is emitted on the first call. This test doesn't check which
// call caused the warning to be emitted.
```

Reply:
Thank you for your suggestion. I updated it.